Fix issue with command dictionary creation

camargo opened the pull request:
Fixes https://github.com/NASA-AMMOS/aerie-ui/issues/650

* Move command dictionary create logic into component
* Make sure command dictionary error clears between sessions
* Update create command dictionary request
* Reset expansion stores when needed

camargo committed:
Fix issue with command dictionary creation

* Move command dictionary create logic into component
* Update create command dictionary request
* Reset expansion stores when needed

duranb reviewed (approved):
Can confirm that the error no longer persists after re-log! Looks good!

Not a critique, but just a question about an observation, is there a reason for moving the toast actions out of the `effects` file and into the component itself?

camargo commented:
> Not a critique, but just a question about an observation, is there a reason for moving the toast actions out of the effects file and into the component itself?

The reason is because I moved the try-catch to the component, and the fail toast is called in the catch so it came with it. I moved the try-catch because we are only calling that create function in the dictionary component, and it is less-buggy to just catch the errors in the component and set error state accordingly (including the toast). We then only depend on local state and don't have to worry about cleaning out the store when the component get's destroyed.
